.changeset/wild-wombats-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'@contentful/f36-forms': patch
+---
+
+Consume density context on Checkbox, Radio components
```

Comment:
I'd love to make the message more explicit, what about: 

```suggestion
Add high-density support to Checkbox and Radio components.
```

Reply:
Ah maybe the revert put the previous message back 👀

Reply:
Good shout, updated!